src/iterative_sorting/iterative_sorting.py

```diff
@@ -1,30 +1,58 @@
 # TO-DO: Complete the selection_sort() function below 
 def selection_sort( arr ):
     # loop through n-1 elements
+    if len(arr) < 2:
+        return arr
     for i in range(0, len(arr) - 1):
         cur_index = i
         smallest_index = cur_index
         # TO-DO: find next smallest element
         # (hint, can do in 3 loc) 
+        for j in range(cur_index, len(arr)):
+            if arr[j] < arr[smallest_index]:
+                smallest_index = j
```

Comment:
Nice work using a second inner for loop. Using the bookshelf analogy, you are scanning each book to the right of the current book, checking to see if it's value is smaller than the current smallest index. Cool!